functests/1_performance/netqueues.go

```diff
@@ -357,3 +361,9 @@ func getRandomDevice(devices map[string][]int) string {
 	}
 	return device
 }
+
+func getPerformanceProfile(profileList *performancev2.PerformanceProfileList) string {
```

Comment:
I think we should not overwrite the argument. The function should read like
```
func getPerformanceProfile() string {
        profileList := performancev2.PerformanceProfileList{}
	err := testclient.Client.List(context.TODO(), &profileList)
	Expect(err).ToNot(HaveOccurred())
	return profileList.Items[0].Name
}
```
what if there are more than one profile? or no profiles? should we handle these cases?

Reply:
i am not sure either,  but in the case of multiple profiles , we can pick the first profile. If there is no profile i can skip the test ?

Reply:
With multiple profiles we should pick any (the first returned is fine), but we should add a log line
With no profiles, logging + skipping the test seems a fine thing to do

Reply:
okay will update the patch